readyplayerme/rpm-unreal-sdk
# [766] Feature/fix cac errors

## [766](https://ready-player-me.atlassian.net/browse/766)

## Description

-   This PR includes a fix that handles the asset id as a string rather than a number



## How to Test

-   run the CAC (make sure you have set subdomain and appID) 
- After you get to the customization screen check the modular outfits and check that all are equipable and there are no errors in the logs about json types like in the image below

![image](https://github.com/readyplayerme/rpm-unreal-sdk/assets/7085672/5313f1ea-0e91-4f86-942d-3ae3c0de01ef)



## Checklist


## QA Testing

-   Mention any useful test cases for this feature and add to QA Testing documentation.

## Details

-   If more details are necessary to support the description with images and videos add them here.
